Formula/nginx.rb

```diff
@@ -145,7 +145,11 @@ def caveats
   end
 
   service do
-    run [opt_bin/"nginx", "-g", "'daemon off;'"]
+    if OS.linux?
+      run [opt_bin/"nginx", "-g", "'daemon off;'"]
+    else
+      run [opt_bin/"nginx", "-g", "daemon off;"]
+    end
```

Comment:
```suggestion
    run macos: [opt_bin/"nginx", "-g", "daemon off;"],
           linux:    [opt_bin/"nginx", "-g", "'daemon off;'"]
```

Reply:
https://github.com/Homebrew/brew/commit/4cf365f6c90aa2e00c6c8b6af8994037696db5ca

Not yet in stable tag.

Reply:
let's maybe revert first and we can improve the syntax later.